types/rox-browser/index.d.ts

```diff
@@ -0,0 +1,273 @@
+  getValue(): BasicType<T>;
+
+  // Unlock the Variant value from changes from the last time it was freezed
+  unfreeze(): void;
+}
+
+/**
+ * manages a remote configuration setting with a value of type string, boolean, or number.
+ * The constructor sets the default value for the remote configuration setting
+ *
+ * https://support.rollout.io/docs/javascript-browser-api#section--rox-configuration-
+ */
+export class Configuration<T extends number | boolean | string> {
+  constructor(defaultValue: T);
+
+  // The name of the Configuration
+  readonly name: string;
+
+  // Default value of the Configuration
+  readonly defaultValue: BasicType<T>;
+
+  // Returns the current value of the Configuration, accounting for value overrides
+  getValue(): BasicType<T>;
+
+  // Unlock the Configuration value from changes from the last time it was freezed
+  unfreeze(): void;
+}
+
+/**
+ * Ensure that TypeScript properly types things with a basic type.
+ * For example, if T is true, returnedtype shall be boolean, not true
+ */
+type BasicType<T> = T extends boolean ? boolean : T extends number ? number : T extends string ? string : never;
+
+/**
+ * Override: Should only be used for development purposes (QA - Feature dev - e2e)
+ *
+ * When you override an existing flag value using the Rox.overrides.setOverride method,
+ * the SDK will disregard existing configuration coming from the dashboard and will
+ * serialize the override on disk this value will be loaded and override the flag
+ * right after you call Rox.setup. To clear the override from the cache you need to
+ * call the Rox.overrides.clearOverride method
+ *
+ * https://support.rollout.io/docs/javascript-browser-api#section--rox-overrides-
+ */
+export interface RoxOverrides {
+  /**
+   * Sets an override value on a specific flag, this function accepts two parameters flag name (
+   * full flag name including namespace) and desired value (from type String).
+   * This function also saves the override value on the local device disk,
+   * so it is "remembered" for the next the SDK is loaded to production.
+   *
+   * https://support.rollout.io/docs/javascript-browser-api#section--rox-overrides-setoverride-
+   *
+   * Note that for boolean flag we still give the value as a string.
+   */
+  setOverride(nameSpacedFlagName: string, value: string): void;
+
+  /**
+   * Clears the override value from the flag (and the disk).
+   *
+   * https://support.rollout.io/docs/javascript-browser-api#section--rox-overrides-clearoverride-
+   */
+  clearOverride(nameSpacedFlagName: string): void;
+
+  /**
+   * Clears all override values
+   */
+  clearAllOverrides(): void;
+
+  getOriginalValue(nameSpacedFlagName: string): string;
+
+  /**
+   * full flag name including namespace
+   *
+   * https://support.rollout.io/docs/javascript-browser-api#section--rox-overrides-hasoverride-
+   */
+  hasOverride(nameSpacedFlagName: string): boolean;
+}
+
+export const overrides: RoxOverrides;
```

Comment:
Instead of doing `export const` and defining some type to describe the available methods, I'd suggest doing the following:
```ts
export namespace overrides {
  ...
  export function clearAllOverrides(): void;
  ...
}
```

Reply:
Indeed, thanks!
Change done.